packages/fx-core/src/common/tools.ts

```diff
@@ -466,3 +466,13 @@ export async function askSubscription(
   }
   return ok(resultSub);
 }
+
+// Determine whether feature flag is enabled based on environment variable setting
+export function isFeatureFlagEnabled(featureFlagName: string): boolean {
+  const flag = process.env[featureFlagName];
+  // can enable feature flag by set environment variable value to "1" or "true"
```

Comment:
Why do we need to set flag to "true" or "1"? Can we use one principle?

Reply:
Provide more ways to ensure our stakeholders can set the environment variable correctly if they didn't go through our instructions carefully.